### DISCUSSION TITLE
Delegating Meeting Management

### DISCUSSION THREAD

**Discussion by furick1** [code4recovery/12-step-meeting-list#1374] [General]
1. Is it possible in WordPress when using the AA mtg plugin to delegate a meeting manager to be able to manage all meetings but not edit the rest of the website?
2. How can we allow local meeting organizers to manage their own meetings so they can update meeting details?
3. If the prior two questions are not possible with granular security of some kind, how are meetings best managed in order to delegate mtg management responsibility and create efficiencies for day of updates? Is a separate form for new meetings and updating existing meetings needed and there is some automated or manual import to replace/update mtgs via a CSV?
4. Is there a setting to not push/share meetings with the AA mtg guide app?

**Comment by joshreisner** [comment]
> Is it possible in WordPress when using the AA mtg plugin to delegate a meeting manager to be able to manage all meetings but not edit the rest of the website

to edit a meeting, users must have the "author" role or above in wordpress. it might be possible to restrict users to edit only the meetings / locations / groups post types with the [User Role Editor](https://wordpress.org/plugins/user-role-editor/) plugin, i'm not sure offhand

> How can we allow local meeting organizers to manage their own meetings so they can update meeting details?

some large areas create user accounts for their districts, it's not typical for sites to create logins for their individual meetings. that said, you can do that, by giving them "author" role or above. if you give a user an author role they can only edit their own meetings, so you need to set the author field of those meetings to them.

> how are meetings best managed in order to delegate mtg management responsibility and create efficiencies for day of updates

there are other methods available. some areas import meetings from a district, which can be managed on a remote website or on a google sheet. for the most part, many sites will have a technical person responsible for technical aspects of site admin, and a program person (usually a central office administrator) who is responsible for day-to-day updates to the listings. many sites use the "feedback email" feature on the settings page to get feedback

> Is there a setting to not push/share meetings with the AA mtg guide app

no. if GSO has connected to your site, they will import all the meetings. if there is a meeting you don't want to propagate, such as a business meeting, we recommend listing this outside the meeting plugin

**Reply by furick1** [reply]
thank you @joshreisner for the quick and on point answers.  Any reason this plugin can't be used for non AA mtgs in WordPress as long as its not for profit? I'm assuming with the mtg type label customization support that it could be used for other purposes and not detract from the AA community?

**Reply by joshreisner** [reply]
the only issue is just that the plugin is designed for listing all one program (singleness of purpose), so it might be a little challenging to list multiple programs in a way that's clear to users. but theoretically yes you should be able to do that by customizing types

meetings will not propagate to the app automatically unless you are an AA service entity that is approved by the AA General Service Office, so nothing to worry about there.

**Reply by furick1** [reply]
one program with multiple mtg types will suffice.  thanks again.

**Reply by Tech4TriCounty** [reply]
Oh and here’s the plugin

https://memberpress.com/

**Comment by Tech4TriCounty** [comment]
I found a plugin that will let you create and manage roles for admin blocks.  I don’t know how fool proof it is. I am granting districts access to district specific pages and I was able to lock them out of everything but the TSML without having to buy the pro version. I’m not sure what makes tsml special or different. It will definitely work if I buy the pro version but they want $79 a year for it. That’s a hard sell for me. If anyone has tried anything else I’d love to hear about it. 

I know this topic was closed but I thought this might help anyone else searching for a solution.

**Reply by schedules000** [reply]
What is the plugin?
